.github/scripts/config.sh

```diff
@@ -80,4 +80,11 @@ PATH="$PATH:$TOOLCHAIN_PATH/bin"
 if [[ "$CCACHE" = 1 ]]; then
     PATH=$CCACHE_LIB_DIR:$TOOLCHAIN_CCACHE_LIB_DIR:$PATH
     export CCACHE_DIR=$CCACHE_DIR_PATH
+    set +o pipefail
+    if ccache -sv 2>&1 | grep -q "invalid option"; then
```

Comment:
I'd move this to https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/.github/scripts/enable-ccache.sh

Reply:
Changed.